tests/cli/RunCest.php

```diff
@@ -354,6 +354,13 @@ public function runIncompleteGherkinTest(CliGuy $I)
         $I->seeInShellOutput('Step definition for `I have only idea of what\'s going on here` not found in contexts');
     }
 
+    public function runFailingGherkinTest(CliGuy $I)
```

Comment:
This test should be named `reportsCorrectFailedStep` to explain what it is testing.